tests/e2e/chain_init/main.go

```diff
@@ -39,7 +41,7 @@ func main() {
 		panic(err)
 	}
 
-	createdChain, err := chain.Init(chainId, dataDir, valConfig, votingPeriod)
+	createdChain, err := chain.Init(chainId, dataDir, valConfig, votingPeriod, forkHeight)
```

Comment:
Assuming that such logic exists, If I were to spin up a single node for state-sync when a chain is already running, what should I be specifying for `forkHeight`?

Reply:
Makes sense, thank you